torchvision/models/googlenet.py

```diff
@@ -56,25 +60,25 @@ def __init__(self, num_classes=1000, aux_logits=True, transform_input=False, ini
 
         self.inception5a = Inception(832, 256, 160, 320, 32, 128, 128)
         self.inception5b = Inception(832, 384, 192, 384, 48, 128, 128)
-        if aux_logits:
-            self.aux1 = InceptionAux(512, num_classes)
-            self.aux2 = InceptionAux(528, num_classes)
+
+        self.aux1 = InceptionAux(512, num_classes)
```

Comment:
While I'm ok with this change (specially that the default value of `aux_logits` was true), this is a BC-breaking change because previously saved state_dicts won't work out of the box anymore if the user had saved a dict with `aux_logits=False`.

I think a solution would be to modify `load_state_dict` for this model to handle this.

@soumith do you think this is necessary?

Reply:
There isnt't a official release that includes GoogLeNet yet, so this shouldn't be much of a concern?

Users could still load their model by passing `strict=False` to `load_state_dict` I think.

Reply:
Actually you are right, googlenet was added after 0.2.2 was released, so this is fine